packages/vscode-lit-plugin/package.json

```diff
@@ -122,13 +122,16 @@
 			"dependencies": [
 				"../lit-analyzer:build"
 			],
-			"command": "node esbuild.script.js",
+			"command": "node esbuild.script.mjs",
 			"files": [
 				"src/**/*",
-				"esbuild.script.js"
+				"../ts-lit-plugin/src/**/*",
+				"../ts-lit-plugin/index.js",
```

Comment:
@aomarks is this kosher, or should I put this script at the toplevel of the monorepo?

Reply:
Should be fine, input files can reach out of the package. It's only output files that currently can't.